projects/vdk-plugins/vdk-structlog/tests/test_plugin.py

```diff
@@ -1,33 +1,157 @@
 # Copyright 2021-2023 VMware, Inc.
 # SPDX-License-Identifier: Apache-2.0
 import os
-import pathlib
 from unittest import mock
 
 from click.testing import Result
 from vdk.plugin.structlog import structlog_plugin
-from vdk.plugin.test_utils.util_funcs import cli_assert_equal
 from vdk.plugin.test_utils.util_funcs import CliEntryBasedTestRunner
 from vdk.plugin.test_utils.util_funcs import jobs_path_from_caller_directory
 
-"""
-This is a sample test file showing a recommended way to test new plugins.
-A good way to test a new plugin is how it would be used in the command that it extends.
-"""
-
-
-def test_dummy():
-    with mock.patch.dict(
-        os.environ,
-        {
-            # mock the vdk configuration needed for our test
-        },
-    ):
-        # CliEntryBasedTestRunner (provided by vdk-test-utils) gives a away to simulate vdk command
-        # and mock large parts of it - e.g passed our own plugins
-        runner = CliEntryBasedTestRunner(structlog_plugin)
-
-        result: Result = runner.invoke(
-            ["run", jobs_path_from_caller_directory("job-using-a-plugin")]
-        )
-        cli_assert_equal(0, result)
+JOB_NAME = "job-with-bound-logger"
+BOUND_TEST_KEY = "bound_test_key"
+BOUND_TEST_VALUE = "bound_test_value"
+EXTRA_TEST_KEY = "extra_test_key"
+EXTRA_TEST_VALUE = "extra_test_value"
+EXCLUDED_BOUND_TEST_KEY = "excluded_bound_test_key"
+EXCLUDED_BOUND_TEST_VALUE = "excluded_bound_test_value"
+
+STOCK_FIELDS = [
+    "level",
+    "file_name",
+    "line_number",
+    "vdk_job_name",
+]  # TODO: add timestamp once bug is resolved
+STOCK_FIELD_REPRESENTATIONS = {
+    "console": {
+        "level": "[INFO ]",
+        "file_name": "10_dummy.py",
+        "line_number": ":23",
+        "vdk_job_name": JOB_NAME,
+    },
+    "json": {
+        "level": '"level": "INFO"',
+        "file_name": '"filename:":"10_dummy.py"',
+        "line_number": '"lineno": 23',
+        "vdk_job_name": f'"vdk_job_name": "{JOB_NAME}"',
+    },
+}
+
+
+def test_structlog_console():
+    for log_format in [
```

Comment:
Please split the test into arrange, test, verity and move most of the arrange in helper method. (https://java-design-patterns.com/patterns/arrange-act-assert/)

Reply:
Also if you want to run same test with multiple parameters, the way to do is it to use `@pytest.mark.parametrize` see https://docs.pytest.org/en/7.3.x/how-to/parametrize.html